docs/documentation.md

```diff
@@ -3,89 +3,87 @@ layout: page
+   1. Install [Docker](https://www.docker.com/products/docker) (>1.10) on your local machine.
+   2. Simply run the following command in the **cloudberry** folder to create an AsterixDB cluster locally.
 
-   - Install [Docker](https://www.docker.com/products/docker) (>1.10) on your local machine.
-   - Simply run the following command in the `cloudberry` folder to create an AsterixDB cluster locally.
-   ```
-   ./script/dockerRunAsterixDB.sh
-   ```
+```
+./script/dockerRunAsterixDB.sh
+```
 
-4. Ingest 324,000 sample tweets into AsterixDB.
+* Ingest 324,000 sample tweets into AsterixDB
 
 ```
 ./script/ingestTwitterToLocalCluster.sh
 ```
 
-5. Ingest 33,107 US population data into AsterixDB (assume you are still in the `cloudberry` folder)
-   - Run the following commands to copy population data into nc1 (which is one of the AsterixDB clusters you just created in the docker)
-      ```bash
-      docker cp noah/src/main/resources/population/adm/allStatePopulation.adm nc1:/home
-      ```
-      ```bash
-      docker cp noah/src/main/resources/population/adm/allCountyPopulation.adm nc1:/home
-      ```
-      ```bash
-      docker cp noah/src/main/resources/population/adm/allCityPopulation.adm nc1:/home
-      ```
-    - Open `noah/src/main/resources/population/sqlpp/ingestPopulation.sqlpp` using a text editor and copy all of its contents. Based on your nc1 IP address and the target path you copied population data into in the last step, you may need to change a little bit of the query, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries. You can check the IP address of nc1 by running:
-      ```bash
-      docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nc1
-      ```
-    - Go to `http://localhost:19001/ `. Copy and paste all the queries from the last step and click `run` with the default settings `SQL++` to ingest data into AsterixDB. After that, you should see `Success: Query Complete` in the output.
+* Ingest 33,107 US population data into AsterixDB (assume you are using Docker):
+* Run the following commands in the **cloudberry** folder to copy three population datasets into nc1
+
+```
+docker cp noah/src/main/resources/population/adm/allStatePopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCountyPopulation.adm nc1:/home
+docker cp noah/src/main/resources/population/adm/allCityPopulation.adm nc1:/home
+```
+
+* Open `noah/src/main/resources/population/sqlpp/ingestPopulation.sqlpp` using a text editor and copy all of its contents. Based on the IP address of nc1 and the path you copied population data into in the last step, you may need to modify the query a little bit, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries. You can check the IP address of nc1 by running:
```

Comment:
Too many words for little information.
`Based on the IP address of nc1 and the path you copied population data into in the last step, you may need to modify the query a little bit, but in most of the cases you don't need to. Detailed instructions can be found in the comments of the queries.`
==> `You may need to change the IP address of  nc1 in the script.`